src/include/txn.h

```diff
@@ -240,6 +240,11 @@ struct __wt_txn_op {
 			} mode;
 		} truncate_row;
 	} u;
+
+/* AUTOMATIC FLAG VALUE GENERATION START */
+#define	WT_TXN_OP_KEY_REPEATED	0x1u
+/* AUTOMATIC FLAG VALUE GENERATION STOP */
+	uint32_t flags;
```

Comment:
Seems a lot of space used for a single flag. I don't feel strongly about it, seems like this came out of a discussion about algorithm to follow.